signer/signer.go

```diff
@@ -93,6 +96,13 @@ func (s *genericSigner) Sign(ctx context.Context, desc ocispec.Descriptor, opts
 	if !opts.Expiry.IsZero() {
 		signReq.Expiry = opts.Expiry
 	}
+	logger.Debugf("sign request:")
+	logger.Debugf("  ContentType: %v", signReq.Payload.ContentType)
+	logger.Debugf("  Content: %s", string(signReq.Payload.Content))
+	logger.Debugf("  Expiry: %v", signReq.Expiry)
+	logger.Debugf("  SigningTime: %v", signReq.SigningTime)
```

Comment:
nit: usually, the creation time is put above the expiry time. so exchange line 102 and 103 might be better.

Reply:
Updated.